### PR TITLE
adding bsd-3 clause license

### PR DESCRIPTION
**Pull request by ocramleznem**
referring to #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,3 +66,7 @@ We don't want to add extra noise while providing a good code-completion.
 Because they are not all meaningful in a test context. When you write unit tests with atoum, you will likely never call
 internal asserters' methods, even if they are public. They provide support for atoum so it can do its job well. But they
 won't help you in your everyday work.
+
+## License
+
+atoum stubs is released under the BSD-3-Clause License. See the bundled LICENSE file for details.
```
Comment (jubianchi) [inline README.md]: `s/is/are/`

**Review by jubianchi — CHANGES_REQUESTED** [review]

**Comment by ocramleznem** [comment]
ouch :-1:  should be fine now.

**Review by jubianchi — APPROVED** [review]

**Comment by tarlepp** [comment]
Hmm you guys realize that is not october yet :D

**Comment by ocramleznem** [comment]
Just getting things done which no one cares about the whole year..

**Comment by jubianchi** [comment]
@tarlepp any contribution is always good. And we welcome everyone.

Yes, it's not October yet but I'll open a lot more issue for the Hacktoberfest ;)

@marcomenzel if you wish to make this PR count for your score, just close it and we'll reopen it in a few days ;)

**Comment by jubianchi** [comment]
@marcomenzel thanks!

**Pull request merged**